PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py

```diff
@@ -76,9 +88,9 @@
         unbiased = Var("electronID('unbiased')",float,doc="ElectronSeed, pT- and dxy- agnostic BDT (raw) score"),
         ptbiased = Var("electronID('ptbiased')",float,doc="ElectronSeed, pT- and dxy- dependent BDT (raw) score"),
         # Isolation
-        miniPFRelIso_chg = Var("userFloat('miniIsoChg')/pt",float,
+        miniPFRelIso_chg = Var("userFloat('miniIsoChg')",float,
                                doc="mini PF relative isolation, charged component"),
-        miniPFRelIso_all = Var("userFloat('miniIsoAll')/pt",float,
```

Comment:
Are these relative isolation or not ? 
In this last PR the division for the pt is removed

Reply:
Yes, they are rel. iso. even if the name suggests not. So the /pt is no longer needed. This is now done [here](https://github.com/cms-sw/cmssw/pull/34005/files#diff-31eff23ea52a2b619e0192f0f8d2e89b7e1740522a692066a66284fc43d547f5R44).